src/globals/scss/core/_links.scss

```diff
@@ -1,45 +1,66 @@
 @include exports("links") {
-  a:link {
-    color: $govuk-link-colour;
-  }
 
-  a:visited {
-    color: $govuk-link-visited-colour;
-  }
+  .govuk-link {
+    @include govuk-font-regular;
+    @include govuk-focusable-fill;
 
-  a:hover {
-    color: $govuk-link-hover-colour;
-  }
+    // Override the tap highlight colour (the color of the highlight that
+    // appears when a link is tapped on some mobile devices). This is
+    // ever-so-slightly darker than the default.
+    -webkit-tap-highlight-color: rgba($govuk-black, .3);
```

Comment:
I've put a ticket in Trello to revisit this because I'm not convinced it's needed: https://trello.com/c/UeF2Dex3/572-revisit-tap-highlighting-overrides-in-govuk-frontend